polylogue/commands.py

```
@@ -570,15 +585,22 @@ def render_command(options: RenderOptions, env: CommandEnv) -> RenderResult:
+        run_payload["meta"] = dict(getattr(options, "meta") or {})
     if getattr(options, "sanitize_html", False):
         run_payload["redacted"] = True
     add_run(run_payload)
-    return RenderResult(
+    result = RenderResult(
         count=len(render_files),
         output_dir=output_dir,
         files=render_files,
         total_stats=totals,
     )
+    setattr(result, "failed_files", failures)
```

Ruff (0.15.9)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)